tests/test_petab.py

```diff
@@ -382,17 +384,23 @@ def test_flatten_timepoint_specific_output_overrides():
                 "obs2__condition1",
             ],
             OBSERVABLE_FORMULA: [
-                f"observableParameter1_{obs1_1_1_1} + observableParameter2_{obs1_1_1_1}",
-                f"observableParameter1_{obs1_2_1_1} + observableParameter2_{obs1_2_1_1}",
-                f"observableParameter1_{obs1_2_2_1} + observableParameter2_{obs1_2_2_1}",
+                f"observableParameter1_{obs1_1_1_1}"
+                f" + observableParameter2_{obs1_1_1_1}",
+                f"observableParameter1_{obs1_2_1_1}"
+                f" + observableParameter2_{obs1_2_1_1}",
+                f"observableParameter1_{obs1_2_2_1}"
+                f" + observableParameter2_{obs1_2_2_1}",
```

Comment:
:neutral_face:

Reply:
we should have thought of something shorter than "observableParameter1_" :sweat_smile: :see_no_evil: